### ISSUE TITLE
Incorrect data format in annotation file

### ISSUE THREAD

**Issue by suuhou** [ucla-mobility/V2V4Real#6]
Hi, Thank you for open sourcing your excellent work. 
I tried to download the dataset, but found that almost all the data under the GPS tag in the annotation file were wrong. 
The data that should have been saved has all become [**!!python/object/apply:numpy.core.multiarray.scalar**]. 
Is there a problem with my download method?

Looking forward to your reply! Thank you in advance!

**Comment by suuhou** [comment]
Sorry, this seems to be just a readability issue. I can correctly load the required data

**Comment by susufang** [comment]
Hi, I meet the same problem when reading the yaml because of such [!!python/object/apply:numpy.core.multiarray.scalar].
So, I want to know how to solve it?
Thank you!

**Comment by suuhou** [comment]
> Hi, I meet the same problem when reading the yaml because of such [!!python/object/apply:numpy.core.multiarray.scalar]. So, I want to know how to solve it? Thank you!

May I ask if the error occurs when you load the yaml file after running the code. 
For me, I can run the code and read the yaml file in the dataset normally. 
At that time I clicked on the yaml file on my computer (window system) and found a lot of messy codes in it that's why I posted this question thread, but those messy codes don't affect python to read it correctly!
